Title bar of some Flatpak apps use non-Libadwaita icon

Title bar of some Flatpak apps use non-Libadwaita icons, but not in the Appimage versions. This doesn't occur in all Flatpak apps. For example:

Appimage version:

![image](https://github.com/FedoraQt/QAdwaitaDecorations/assets/37974764/5e91925e-3730-464a-834c-df3b575ba5a2)

Flatpak version:

![image](https://github.com/FedoraQt/QAdwaitaDecorations/assets/37974764/045fab5c-c259-4aae-895b-0a58895f8395)

I am inexperienced with building; I have qt5-qtwayland installed, but I get error
```
CMake Error at CMakeLists.txt:28 (find_package):
  Could not find a package configuration file provided by "QT" with any of
  the following names:

    Qt5Config.cmake
    qt5-config.cmake

  Add the installation prefix of "QT" to CMAKE_PREFIX_PATH or set "QT_DIR" to
  a directory containing one of the above files.  If "QT" provides a separate
  development package or SDK, be sure it has been installed.

```

I could also install an rpm if you have one.

As for the debug, 
calibre, with QT_LOGGING_RULES unset:
```
[adonnen@fwlt-fedora ~]$ calibre
qt.qpa.qadwaitadecorations: Failed to find an svg icon for  "window-close-symbolic.svg"
qt.qpa.qadwaitadecorations: Failed to find an svg icon for  "window-minimize-symbolic.svg"
qt.qpa.qadwaitadecorations: Failed to find an svg icon for  "window-maximize-symbolic.svg"
qt.qpa.qadwaitadecorations: Failed to find an svg icon for  "window-restore-symbolic.svg"
```

Calibre with QT_LOGGING_RULES=QT_LOGGING_RULES='qt.qpa.qadwaitadecorations=true':
```
[adonnen@fwlt-fedora ~]$ calibre
qt.qpa.qadwaitadecorations: Using Qt6 version
qt.qpa.qadwaitadecorations: Changing color scheme to  light
qt.qpa.qadwaitadecorations: Failed to find an svg icon for  "window-close-symbolic.svg"
qt.qpa.qadwaitadecorations: Failed to find an svg icon for  "window-minimize-symbolic.svg"
qt.qpa.qadwaitadecorations: Failed to find an svg icon for  "window-maximize-symbolic.svg"
qt.qpa.qadwaitadecorations: Failed to find an svg icon for  "window-restore-symbolic.svg"
qt.qpa.qadwaitadecorations: Changing color scheme to  light
qt.qpa.qadwaitadecorations: Changing titlebar layout to  "appmenu:close"
```

Konsole with QT_LOGGING_RULES unset:
```
[adonnen@fwlt-fedora build]$ konsole
QSocketNotifier: Can only be used with threads started with QThread
kf.windowsystem: Could not find any platform plugin
```

Konsole with QT_LOGGING_RULES='qt.qpa.qadwaitadecorations=true':
```
QSocketNotifier: Can only be used with threads started with QThread
kf.windowsystem: Could not find any platform plugin
qt.qpa.qadwaitadecorations: Using Qt5 version with Qt6 backports
qt.qpa.qadwaitadecorations: Changing color scheme to  light
qt.qpa.qadwaitadecorations: Using  "window-close-symbolic.svg"  from  "breeze"  theme
qt.qpa.qadwaitadecorations: Using  "window-minimize-symbolic.svg"  from  "breeze"  theme
qt.qpa.qadwaitadecorations: Using  "window-maximize-symbolic.svg"  from  "breeze"  theme
qt.qpa.qadwaitadecorations: Using  "window-restore-symbolic.svg"  from  "breeze"  theme
qt.qpa.qadwaitadecorations: Changing color scheme to  light
qt.qpa.qadwaitadecorations: Changing titlebar layout to  "appmenu:close"
```

Kate did not show output either way. Kate and Konsole are both qt5 and both affected.
I included Calibre separately because (as I said above) Calibre might be unaffected by this/have its own error, and is qt6. Are there any other qt6 apps I can test to see if the problem is due to Calibre's custom code?

--- a/src/qadwaitadecorations.cpp
+++ b/src/qadwaitadecorations.cpp
@@ -177,7 +177,10 @@ void QAdwaitaDecorations::updateColors(bool useDarkColors)
 
 QString getIconSvg(const QString &iconName)
 {
-    const QStringList themeNames = { QIcon::themeName(), QIcon::fallbackThemeName() };
+    const QStringList themeNames = { QIcon::themeName(), QIcon::fallbackThemeName(),
+                                     QLatin1String("Adwaita") };
+    qCDebug(QAdwaitaDecorationsLog) << "Icon themes: " << themeNames;
+
     for (const QString &themeName : themeNames) {
         for (const QString &path : QIcon::themeSearchPaths()) {
             if (path.startsWith(QLatin1Char(':')))

